exercises/practice/poker/test/poker_test.exs

```diff
@@ -1,18 +1,34 @@
 defmodule PokerTest do
   use ExUnit.Case
 
+  defp assert_poker(winning_hands, expected_winning_hands) do
+    winning_hands =
+      winning_hands
+      |> Enum.map(&Enum.sort/1)
+      |> Enum.sort()
+
+    expected_winning_hands =
+      expected_winning_hands
+      |> Enum.map(&Enum.sort/1)
+      |> Enum.sort()
+
```

Comment:
The first error you get before having implemented anything is pretty ugly right now:
```
  1) test single hand always wins (PokerTest)
     test/poker_test.exs:19
     ** (Protocol.UndefinedError) protocol Enumerable not implemented for nil of type Atom. This protocol is implemented for the following type(s): Range, HashSet, HashDict, GenEvent.Stream, IO.Stream, File.Stream, Function, List, Stream, Map, Date.Range, MapSet
     code: assert_poker(winning_hands, [high_of_jack])
     stacktrace:
       (elixir 1.11.3) lib/enum.ex:1: Enumerable.impl_for!/1
       (elixir 1.11.3) lib/enum.ex:141: Enumerable.reduce/3
       (elixir 1.11.3) lib/enum.ex:3473: Enum.map/2
       test/poker_test.exs:7: PokerTest.assert_poker/2
       test/poker_test.exs:22: (test)
```

Let's make it nicer, like this:
```
 1) test single hand always wins (PokerTest)
     test/poker_test.exs:24
     Assertion with == failed
     code:  assert winning_hands == expected_winning_hands
     left:  nil
     right: [["4S", "5S", "7H", "8D", "JC"]]
     stacktrace:
       test/poker_test.exs:27: (test)
```

By doing this simple code change:

```suggestion
    winning_hands =
      if winning_hands do
        winning_hands
        |> Enum.map(&Enum.sort/1)
        |> Enum.sort()
      end

    expected_winning_hands =
      if expected_winning_hands do
        expected_winning_hands
        |> Enum.map(&Enum.sort/1)
        |> Enum.sort()
      end
```